pages.fr/common/ansiweather.md

```diff
@@ -0,0 +1,16 @@
+# ansiweather
+
+> Un script Shell pour afficher les conditions météo actuelles dans votre terminal.
+> Plus d'informations : <https://github.com/fcambus/ansiweather>.
+
+- Affiche une prévision avec le système métrique pour les 5 prochains jours dans la ville de Paris, France :
+
+`ansiweather -u {{metric}} -f {{5}} -l {{Paris,FR}}`
```

Comment:
```suggestion
`ansiweather -u {{métrique}} -f {{5}} -l {{Paris,FR}}`
```

Reply:
Hi ! 
This time `métrique` is the correct translation for `metric` :p 

But according to the documentation of ansiweather, the `-u` option only take 2 possible values
![image](https://user-images.githubusercontent.com/9024389/179672108-f858db01-bd2b-4b54-b2b3-23d4e71e9731.png)

So i think this time we should not translate this placeholder either 😆

Reply:
I agree. While `métrique` may be the correct French translation of `metric`, `metric` holds syntactical meaning here so ti cannot be translated without breaking the example.